strawberry/arguments.py

```diff
@@ -152,9 +152,9 @@ def convert_argument(
     if isinstance(type_, LazyType):
         return convert_argument(value, type_.resolve_type(), scalar_registry, config)
 
-    if hasattr(type_, "_type_definition"):  # TODO: Replace with StrawberryInputObject
-        type_definition: TypeDefinition = type_._type_definition  # type: ignore
-
+    if type_definition := get_type_definition(
```

Comment:
we can't use python 3.8 syntax just yet :)

also can we make get_type_definition "private"? I don't we should expose it just yet to users 😊

Reply:
Sure